content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -65,7 +65,6 @@ to enable the DynamicKubeletConfiguration feature.
 
 {{< tabs name="tab-kubelet" >}}
 {{< tab name="kubelet" include="generated/kubeadm_alpha_kubelet.md" />}}
-{{< tab name="download" include="generated/kubeadm_alpha_kubelet_config_download.md" />}}
```

Comment:
@prasadkatti ,
If you have time, would you double check that "enable-dynamic" is including the correct file?
I see a generated file, `generated/kubeadm_alpha_kubelet_config_enable-dynamic.md`

Reply:
You are right. It is including the wrong file. I can fix that.